Box_intersection_d/include/CGAL/Box_intersection_d/segment_tree.h

```diff
@@ -339,6 +338,7 @@ void segment_tree( RandomAccessIter1 p_begin, RandomAccessIter1 p_end,
     const T sup = box_limits< T >::sup();
 
 #if CGAL_BOX_INTERSECTION_DEBUG
+  CGAL_STATIC_THREAD_LOCAL_VARIABLE(int, level, -1);
```

Comment:
`level` is used for debugging only. it being a thread-local variable is the right choice: anyway the `Box_intersection_d` is not thread-safe if the same data is concurrently used by multiple threads.